README.md

```diff
@@ -58,48 +58,49 @@ end
+| cmake       | cmake-language-server                                                                                                          |
+| cpp         | clangd                                                                                                                         |
+| csharp      | OmniSharp                                                                                                                      |
+| css         | css-language-features (pulled directly from the latest VSCode release)                                                         |
+| dockerfile  | docker-langserver                                                                                                              |
+| elixir      | Elixir Language Server (elixir-ls)                                                                                             |
+| elm         | Elm Language Server (elm-ls)                                                                                                   |
+| ember       | Ember Language Server                                                                                                          |
+| fortran     | Fortran Language Server (fortls)                                                                                               |
+| go          | gopls                                                                                                                          |
+| graphql     | GraphQL language service                                                                                                       |
+| haskell     | haskell-language-server                                                                                                        |
+| html        | html-language-features (pulled directly from the latest VSCode release)                                                        |
+| java        | Eclipse JDTLS with Lombok                                                                                                      |
+| json        | json-language-features (pulled directly from the latest VSCode release)                                                        |
+| kotlin      | kotlin-language-server                                                                                                         |
+| latex       | texlab                                                                                                                         |
+| lua         | (sumneko) lua-language-server                                                                                                  |
+| php         | intelephense                                                                                                                   |
+| puppet      | puppet-languageserver (pulled directly from the latest [GitHub][https://github.com/puppetlabs/puppet-editor-services] release) |
```

Comment:
please replace `puppet-languageserver (pulled directly from the latest [GitHub][https://github.com/puppetlabs/puppet-editor-services] release)` with just `puppet-editor-services`

Reply:
Fixed this in a new commit, take a look!